test/e2e/run_networking_test.go

```diff
@@ -716,7 +716,32 @@ EXPOSE 2004-2005/tcp`, ALPINE)
 	})
 
 	It("podman cni network works across user ns", func() {
-		SkipUntilAardvark(podmanTest)
+		SkipIfNetavark(podmanTest)
+		netName := stringid.GenerateNonCryptoID()
+		create := podmanTest.Podman([]string{"network", "create", netName})
+		create.WaitWithDefaultTimeout()
+		Expect(create).Should(Exit(0))
+		defer podmanTest.removeNetwork(netName)
+
+		name := "nc-server"
+		run := podmanTest.Podman([]string{"run", "--log-driver", "k8s-file", "-d", "--name", name, "--net", netName, ALPINE, "nc", "-l", "-p", "9480"})
+		run.WaitWithDefaultTimeout()
+		Expect(run).Should(Exit(0))
+
+		// NOTE: we force the k8s-file log driver to make sure the
+		// tests are passing inside a container.
+		run = podmanTest.Podman([]string{"run", "--log-driver", "k8s-file", "--rm", "--net", netName, "--uidmap", "0:1:4096", ALPINE, "sh", "-c", fmt.Sprintf("echo podman | nc -w 1 %s.dns.podman 9480", name)})
```

Comment:
@baude this test passes when I do:

```
$ sudo ./bin/podman run --log-driver k8s-file --rm --net blahblah --uidmap 0:1:4096 alpine sh -c "echo podman | nc -w 1 nc-server.dns.podman 9480"
```

but exits with code `1` when I run make localintegration.

Reply:
actually this comment is meant for the test right below `podman netavark network works across user ns`